components/KebabPopUpController.tsx

```diff
@@ -94,6 +99,7 @@ export function useKebabPopUp(props) {
     );
     otpError = useSelector(service, esignetSelectOtpError);
     walletBindingError = useSelector(service, esignetSelectWalletBindingError);
+    bindingAuthFailedError = useSelector(service, selectBindingAuthFailedError);
```

Comment:
It's redundant

Reply:
I have removed this as part of refactoring